### DISCUSSION TITLE
M4P Music Player

### DISCUSSION THREAD

**Discussion by VickyMerzOwn** [clangen/musikcube#523] [Q&A]
Doesn't musikCube play M4P files? I have some songs in my iTunes library as part of my monthly Apple Music subscription and they are in the M4P format. Can't I play them using musikCube?

**Comment by PythonTryHard** [comment]
A quick search using `m4p ffmpeg` (as afaict, `musikcube` uses `ffmpeg` internally to decode music) shows that M4P files have DRM, which confirmed my hunch since "monthly Apple Music subscription" felt life it had some sort of DRM enabled.
